lib/scoped_search/query_builder.rb

```diff
@@ -485,6 +489,16 @@ def to_sql(builder, definition, &block)
             raise ScopedSearch::QueryNotSupported, "Don't know how to handle this operator node: #{operator.inspect} with #{children.inspect}!"
           end
         end
+
+        private
+
+        def validate_value(field, value)
+          validator = field.validator
+          if field.validator && field.only_explicit
```

Comment:
We can throw an exception only in case the field is referred explicitly. The method will be called from default fields query too, and there we don't need to validate (and we can't throw an exception).

I am not completely sure what kind of validation/indication would you use for default fields query type.